docs/contributing/style_guide.rst

```diff
@@ -24,7 +24,22 @@ code
 * **Experimental methods should log an experimental warning when called**: "Warning: some_method is experimental. Methods, APIs, and core behavior may change in the future."
 * **Experimental classes should log an experimental warning when initialized**: "Warning: great_expectations.some_module.SomeClass is experimental. Methods, APIs, and core behavior may change in the future."
 * **Docstrings are highly recommended**. We use the Sphinx's `Napoleon extension <http://www.sphinx-doc.org/en/master/ext/napoleon.html>`__ to build documentation from Google-style docstrings.
-* **Lint your code**. Our CI system will check using ``black`` and ``isort``. We have a git pre-commit configuration in the repo, so you can just run ``pre-commit install`` to automatically run your changes through the linting process before submitting.
+* **Lint your code**. Our CI system will check using ``black`` and ``isort``.
```

Comment:
I'm not clear on how linting currently works. I haven't been linting, and that doesn't seem to have raised any issues.

Reply:
You probably have been either perfectly writing well-formatted code or are using an IDE that lints magically for you?

Reply:
Either way, I appear to be magical.